packages/mds-provider/tests/status-changes.spec.ts

```diff
@@ -113,6 +113,20 @@ const fixtures: {
 ]
 
 describe('Testing Mapping of Agency Events to Provider Status Changes', () => {
+  before(done => {
+    fixtures.forEach(({ agency, provider }) =>
+      // eslint-disable-next-line no-console
+      console.log(
```

Comment:
Does this need to be a `console.log`? I feel like this would make more sense as a unit test if it was using `assert`, I don't expect this to ever throw an error.

Reply:
It's not intended to be any sort of a test ... I have been asked enough times to generate the mappings that I wanted an easy/permanent way to dump them out. I did something similar with the MDS API scope values/descriptions.